vignettes/hys_law.Rmd

```diff
@@ -13,12 +13,15 @@ knitr::opts_chunk$set(
   comment = "#>"
 )
 
+library(admiral)
```

Comment:
@bms63 

Think @bundfussr  makes excellent suggestions.

Will source template directly, rather than use function `use_ad_template` (either my error or I though the function added error checking)

Reply:
@bms63 

Think @bundfussr  makes excellent suggestions.

Will source template directly, rather than use function `use_ad_template` (either my error or I though the function added error checking)

_Update_
Running in a few issues with the ad_adlb.R,  has several library calls and uses data() to load data;  getting myself fouled up.   The resulting data no longer match the original ...

Reply:
OH that is concerning - what are the mismatches?

Reply:
Let me give u report Monday afternoon.